README.md

```diff
@@ -60,4 +57,70 @@ These tables are an addition to the existing PEtab format to describe the covari
 
 - No table
 
-  If no table is provided, only the variances, but no covariances are estimated.
\ No newline at end of file
+  If no table is provided, only the variances, but no covariances are estimated.
+  
+## Measurement table
+
+In addition to the existing descriptions of fixed effects, the NLME extension adds the following columns:
+
+- `$covariateId` one column per distinct covariate
+  
+## Covariate table
+
+This table describes the addition of covariate table in the PEtab NLME extension.
+
+### Detailed field description
+
+- `covariateId` [STRING]
+
+   ID of the covariate.
+
+- `covariateParameter` [STRING]
+
+   Defines the parameter where the covariate effect (COVEFF) is modelled. It must be a `parameterId` defined in the parameter table. The COVEFF would be multiplied with the `parameterId`.
+
+-  `covariateFormula` [STRING: 'exclude' or 'fractional' or 'power' or free text formula]
+
+   Covariate effect function as plain text formula expression. The user can define its own formula or can use predefined models: 'exclude' or 'fractional' or 'power'.
+   - 'exclude' --> Covariate not included on the parameter.
+   $\text{COVEFF}=1$
+
+   - 'fractional' --> Linear function. For categorical and continuous covariates.
+   
+   For continuous covariates:
+   $\text{COVEFF}= 1 + \theta_{cov} \times \left(\text{covariateId} - median(\text{covariateId})\right)$
+   
+   For categorical covariates:
+   
+   $\text{if }  covariateValue  ==  Most common; \quad  \text{COVEFF} = 1$
+   
+   $\text{else }  ; \quad \text{COVEFF} = ( 1 + \theta_{cov})$
+
+   - 'power' --> Power function. Continuous covariates only.
+   $\text{COVEFF}= {\left(\frac{\text{covariateId}}{median(\text{covariateId})}\right)}^{\theta_{cov}}$
+   
+   Note that free text formula is also allowed
+
+- `covariateType` [STRING]
+
+   Defines whether a covariate is categorical or numerical.
```

Comment:
I am not sure about using strings for categorical variables. Currently most of the pharmacometrics modeling tools adopt numerical values for these. If we use strings, that would add another layer of processing when compiling the model into the corresponding software tool.

But we can discuss that with the rest of the project team.

Moreover another thing I can think of is that sometimes data anonymization may be needed, e.g. to exchange with an external partner, so with that we would be giving the actual, e.g. sex, race, ethnicity, positivity for a given chemical assay... and this could be sometimes problematic.